### ISSUE TITLE
Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by djensenius** [mattpannella/pocket-updater-utility#175]
I'm getting:

```
Update Process Complete
Well, something went wrong. Sorry about that.
Object reference not set to an instance of an object.
```

When using 2.32.0, with jtbeta.zip installed. Any ideas what this might be?

**Comment by mattpannella** [comment]
hrm. but it completes everything?

**Comment by djensenius** [comment]
I think?

**Comment by mattpannella** [comment]
does it happen every time? what os? what settings are you using? is it doing all the steps in update all or did you disable some?

**Comment by djensenius** [comment]
MacOS 14 public beta
I have some cores disabled. I can post my settings if that's helpful.

**Comment by pdecampos** [comment]
also seeing the same issue. i've tried removing the pocket_updater_settings.json file, and trying from fresh and it's still happening.

for me, mainly it happens at the main screen, right after it checks for updates. i was using an older version today without issue. i've also had it happen after running an "update all", right after downloading the pocket firmware.

this is on macOS 13.4.1

**Comment by mattpannella** [comment]
if you guys could paste your settings.json file in here, that'd be helpful in reproducing. thanks

**Comment by pdecampos** [comment]
settings.json:

```
{
  "firmware": {
    "version": ""
  },
  "config": {
    "download_assets": true,
    "archive_name": "openFPGA-Files",
    "github_token": null,
    "download_firmware": true,
    "core_selector": false,
    "preserve_platforms_folder": true,
    "delete_skipped_cores": true,
    "download_new_cores": "ask",
    "build_instance_jsons": true,
    "crc_check": true,
    "fix_jt_names": true,
    "skip_alternative_assets": false,
    "use_custom_archive": false,
    "custom_archive": {
      "url": "https://updater.retrodriven.com",
      "index": "updater.php"
    }
  },
  "coreSettings": {
    "agg23.Arduboy": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.GameAndWatch": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.NES": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.PC Engine": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.PokemonMini": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.Pong": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.SNES": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.Tamagotchi_P1": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "agg23.WonderSwan": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "antongale.exerion": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "antongale.performan": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "antongale.slapfight": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "antongale.taitosj": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.Asteroids": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.Dominos": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.DonkeyKong": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.LunarLander": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.QBert": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.RadarScope": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.SpaceRace": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.SuperBreakout": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtbtiger": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtcastle": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtcomsc": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtcontra": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtcop": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtcps1": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtdd": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtdd2": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtexed": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtgng": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtgunsmk": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtkarnov": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtkicker": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtkiwi": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtkunio": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtmikie": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtoutrun": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtpang": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtpinpon": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtrastan": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtroadf": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtroc": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtsarms": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtsbaskt": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtsectnz": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtsf": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jttrack": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jttrojan": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtvigil": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtvulgus": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "jotego.jtyiear": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Mazamars312.Amiga": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Mazamars312.NeoGeo": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "Mazamars312.NeoGeo_Overdrive": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Mazamars312.PC Engine CD": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "nullobject.tecmo": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.bankpanic": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.congo": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.digdug": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.galaga": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.gberet": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.pooyan": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.supervision": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "boogermann.xevious": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "ericlewis.Genesis": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "pram0d.bakraid": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "pram0d.batrider": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "pram0d.garegga": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "pram0d.kingdmgp": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "pram0d.snowbros2": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "pram0d.sstriker": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "somhi.flappybird": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spacemen3.PDP1": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.2600": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.7800": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Adventurevision": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Arcadia": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.ChannelF": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Coleco": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Creativision": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.GB": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.GBA": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.GBC": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.GG": {
      "skip": false,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Gamate": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Gameking": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Genesis": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Intv": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Megaduck": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.NES": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Odyssey2": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.SG-1000": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.SMS": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Studio2": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.SuperGB": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.Supervision": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    },
    "Spiritualized.VideoBrain": {
      "skip": true,
      "download_assets": true,
      "platform_rename": true
    }
  }
}
```

**Comment by Redinhead** [comment]
Udate. Whenever I got the error message, I would try inserting a number in exe, like choosing a number from the menu, then press enter and the .exe would close down. After a few times with this outcome, I opened it up and the message was the correct one. "choose your destiny:". No Idea what I did differently

**Comment by mattpannella** [comment]
thanks guys. i found the issue

**Comment by mattpannella** [comment]
this should be fixed in 2.32.1
